spec/features/idv/steps/gpo_step_spec.rb

```diff
@@ -103,12 +106,38 @@
         travel_to(days_passed.days.from_now) do
           sign_in_live_with_2fa(user)
           visit idv_request_letter_path
+
           expect(page).to have_current_path(idv_verify_by_mail_enter_code_path)
-          expect(page).not_to have_css('.usa-button', text: t('idv.buttons.mail.resend'))
+          expect(page).not_to have_link(
+            t('idv.gpo.did_not_receive_letter.intro.request_new_letter_link'),
+          )
         end
       end
     end
 
+    context 'too little time has passed', :js do
+      it 'does not present the user the option to to resend', :js do
```

Comment:
in the interest of test performance, is it feasible to set the test up so we don't simulate the whole IDV process for each of these tests?

Reply:
Will look into this.

Reply:
Inlined a bunch of the rate-limiting tests into the happy-path test.